api/routes/urls.js

```diff
@@ -0,0 +1,179 @@
+const { StatusCodes } = require("http-status-codes");
+const router = require("express").Router();
+
+const {
+  getAllUrls,
+  addUrl,
+  getUrlByShorterUrl,
+  modifyUrl,
+  deleteUrl,
+  getShortUrlsStartingBy,
+  getShortUrlsContaining,
+  getShortUrlsNotContaining,
+} = require("../services/urlServices");
+const {
+  checkConnectionToDb,
+  createDbConnection,
+  disconnectFromDb,
+} = require("../database/connection");
+
+router.get("/all", async (req, res) => {
+  const connection = createDbConnection();
```

Comment:
can do the connection as a regular var and not const, just for initialization, and declare the value un the try catch